src/auth/auth.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller, Req, Post, UseGuards, Body } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+import { RegisterInput } from './dto/register.dto';
+import { Request } from 'express';
+import { User } from 'src/entities/user.entity';
```

Comment:
Let's briefly discuss in our meeting about the point you mentioned regarding the location for entity definition. 
CC @Saad-Amjad @asad-mlbd 

> Entities are placed in a root folder inside src
This is because entities might be needed in more than one module